blog/_posts/2017-09-23-RGSoC- the grand finale.md

```diff
@@ -0,0 +1,99 @@
+
+## Things learnt:
+
+* Learnt to work with docker.
+
+* Learnt Git basics and a lot more.
+
+* Worked on more features in Ruby on Rails.
+
+* Got a good grip in Ruby.
+
+* Learnt a lot of new terms and concepts.
+
+* Time management.
+
+* Patience.
+
+* Being regular with updates.
+
+## Issues worked on:
+
+This summer taught us a lot about the workflow of a project and how priorities change. We started with three issues in the beginning of the summer and now, towards the end, our plan has more or less changed. We integrated the site with it's MailChimp account. This feature interested us a lot because we experimented around with MailChimp and actually sent out a few promotional e-mails(from a dummy account). Now we know how all the popular websites send out all those emails :) 
+Implementing it took a lot of time because with every single line of code we wrote we learnt so much. 
+
+We also took up the job of porting the project to Bootstrap 3 and making the pages mobile-responsive. That helped us go through the whole project page-by-page giving us a lot of insight into how a Rails application works. 
+
+In the midst of this, we solved many minor issues in the repo. 
+We lodged a few issues, cleaned up the documentation by relating to the minor difficulties we had in setting up the project on our systems.  
+
+Our next major feature is to implement an Attendance system. Still in the process and loving every moment of it. 
+
+## Problems faced:
+
+<img src="/img/blog/2017/confused.gif" alt="Thankyou"/>
+<div align="center" class="image-credits">credits:giphy.com</div>
+
+In the initial weeks, we were troubled a lot by Docker issues. We were completely new to the platform and faced difficulty in setting it up. Fixing the issues one by one helped us learn in-depth about Docker. We went through the documentation of Docker a lot of times to figure out the problems we were facing. It really made us appreciate the usefulness of Docker in production. 
+
+ In the middle of the SoC, both of our laptops(simultaneously !) crashed due to a kernel version error and hardware issues(keyboard stopped responding). Then came up another error, due to which our console stopped working. Googling about the issue brought us nowhere, everything got messed up. At last with the help of our coaches, we were able to fix the error. The funniest thing is that it was just the matter of changing 2-3 lines of code. 
+
+## What’s next?
+
+We would be continuing our work pattern, working on our skills, exploring new areas, mentoring our juniors. We are planning to attend the [GHCI](https://ghcindia.anitaborg.org/) and [dotCSS, dotJS 2017](https://www.dotcss.io), attend regular meetups of our local community, encouraging more people to take up the work. We are in the process of organising a Rails Girls workshop in our city and we have a lot of support from it by various organizations. 
```

Comment:
I think even though dotCSS(https://www.dotcss.io/) and dotJS(https://www.dotjs.io/) are organized together, they are 2 different conferences. Can you update the link(s) separately of the conference(s) you'll be attending?

Reply:
@nerdyvaishali We got tickets for both conference. The invitation was titled dotCSS and dotJS.

Reply:
@Anagha22 That I know. But as you can see in my comment the links are different. So, it would be good to have both redirecting links instead of one.

Reply:
@nerdyvaishali Sorry, I misinterpret that. Will update it now.